microsoft-edge/dualengine/concepts/launching-internet-explorer.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Launching Internet Explorer
+description: Launching Internet Explorer for DualEngine API Use
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+
+# Launching Internet Explorer
+Once you have built your adapter DLL you must launch Internet Explorer such that it is aware that it is being used
+for the DualEngine API. This article will go over the steps required to do this.
+
+## Required Command Line Arguments
+The following command line arguments are required to be present in order to launch Internet Explorer in the correct
+way to use the DualEngine

```

Comment:
```suggestion
way to use the DualEngine.
```

Reply:
The following command-line arguments must be present in order to launch Internet Explorer in the correct way to use the DualEngine API.